code/modules/vending/_vending.dm

```diff
@@ -715,7 +715,7 @@ GLOBAL_LIST_EMPTY(vending_products)
 				.["user"]["department"] = "No Department"
 	.["stock"] = list()
 	for (var/datum/data/vending_product/R in product_records + coin_records + hidden_records)
-		.["stock"][R.name] = R.amount
+		.["stock"]["[replacetext(replacetext("[R.product_path]", "/obj/item/", ""), "/", "-")]"] = R.amount
```

Comment:
Do we have to replace / with -?

Reply:
The path is defined above when passed to TGUI, I'm just using what was there.

Reply:
It's more consistent with HTML standards for IDs to use - instead of / or other formatting, since it's used as the row's key basically in the TGUI